### ISSUE TITLE
When the number of training steps is too much, the loss becomes NaN

### ISSUE THREAD

**Issue by Wxl-stars** [GaParmar/img2img-turbo#118]
Besides modifying the dataset, I used the settings from the paper.
But when the steps up to 8w, the loss becomes NaN. What can I do to fix the problem?
